core/geometry/src/test/curve/Curve.test.ts

```diff
@@ -605,11 +604,12 @@ class ExerciseCurve {
     {
       if (Checker.noisy.testTransitionSpiral) {
         for (const spiral of [
+          IntegratedSpiral3d.createRadiusRadiusBearingBearing(Segment1d.create(0, 1000), AngleSweep.createStartEndDegrees(0, 10), Segment1d.create(0, 1), Transform.createIdentity()),
           DirectSpiral3d.createDirectHalfCosine(Transform.createIdentity(), 100, 300, undefined),
           DirectSpiral3d.createJapaneseCubic(Transform.createIdentity(), 100, 300, undefined),
           DirectSpiral3d.createArema(Transform.createIdentity(), 100, 300, undefined),
-          // TODO: comment out until fix clothoid exerciseClosestPoint bug (1)
-          // IntegratedSpiral3d.createRadiusRadiusBearingBearing(Segment1d.create(0, 1000), AngleSweep.createStartEndDegrees(0, 10), Segment1d.create(0, 1), Transform.createIdentity())
+          //  TODO: comment out until fix clothoid exerciseClosestPoint bug (1)
```

Comment:
Lines 611, 612 are redundant and can be removed. Otherwise, everything looks great. Thanks for fixing these bugs discovered by the new closestPt tests!

Reply:
(corrected)